src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeFile(processingData(readFile(fromFileName)), toFileName);
+    }
+
+    private String processingData(String stringFile) {
+        final int numberAmount = 1;
+        StringBuilder stringBuilder;
```

Comment:
why do you declare this var 14 lines BEFORE first using? **And think about the name.**